medicaltorch/transforms.py

```diff
@@ -390,28 +399,31 @@ def __call__(self, sample):
         # TODO: Use the axis index for factoring this loop
         for x in range(input_data.shape[self.axis]):
             if self.axis == 0:
-                input_rotated[x,:,:] = F.rotate(Image.fromarray(input_data[x,:,:], mode='F'), angle)
+                input_rotated[x, :, :] = F.rotate(Image.fromarray(input_data[x, :, :], mode='F'), angle)
                 if self.labeled:
-                    gt_rotated[x,:,:] = F.rotate(Image.fromarray(gt_data[x,:,:], mode='F'), angle)
+                    gt_rotated[x, :, :] = F.rotate(Image.fromarray(gt_data[x, :, :], mode='L'), angle)
```

Comment:
Not sure: No need to multiply by 255 before here?

Reply:
Hum ... I don't think so as we use gt_data (which has been multiplied by 255 in `__getitem__`). But I might be wrong ?